src/components/GitStars.js

```diff
@@ -0,0 +1,67 @@
+import React from "react"
+import styled from "styled-components"
+import Emoji from "./Emoji"
+import Icon from "./Icon"
+import Link from "./Link"
+
+const Container = styled(Link)`
+  margin-right: -0.5rem;
+  float: right;
+  display: flex;
+  flex-direction: row;
+  text-decoration: none;
+  border-radius: 4px;
+  border: 1px solid ${(props) => props.theme.colors.lightBorder};
+  color: ${(props) => props.theme.colors.text};
+  &:hover {
+    box-shadow: 0 0 1px ${(props) => props.theme.colors.primary};
+  }
+  &:hover path {
+    fill: ${(props) => props.theme.colors.primary};
+  }
+  background: ${(props) => props.theme.colors.lightBorder};
+`
+
+const Pill = styled.div`
+  text-align: center;
+`
+
+const GlyphPill = styled(Pill)`
+  display: flex;
+  flex-direection: row;
```

Comment:
Typo here, but you can just remove this line (`flex-direction` defaults to `row`)